extension/js/common/core/att.ts

```diff
@@ -105,6 +105,8 @@ export class Att {
       return 'encryptedFile';
     } else if (this.name.match(/(cryptup|flowcrypt)-backup-[a-z0-9]+\.(key|asc)$/g)) {
       return 'privateKey';
+    } else if (this.type === 'application/pgp-keys') {
```

Comment:
Interesting this was missing :+1:

Reply:
yes, it was identified as `encryptedMsg` and later the message was analyzed to detect it's a public key.
I decided that we'd better idenitify it as `publicKey` right away

Reply:
Excellent, thanks!